.github/workflows/camel_version.yaml

```diff
@@ -120,25 +111,5 @@ jobs:
           steps.uiTest_Ubuntu.outcome == 'cancelled' ||
           steps.uiTest_MacOS_Windows.outcome == 'cancelled')
         with:
-          name: ${{ matrix.os }}-${{ matrix.version }}-ui-test-screenshots
+          name: ${{ matrix.os }}-${{ matrix.camel-version }}-ui-test-screenshots
           path: test-resources/screenshots/*.png
-
-  check:
```

Comment:
@apupier I am not sure.. do we want to block PR to merge when also some of Camel Version CI runs fails or to check and rely only Main CI is enough?

Reply:
Ideally we should block PR merge.

Reply:
ok, added job to check also camel version pipeline and block PR in case of failures